packages/playground/src/views/farms.vue

```diff
@@ -145,15 +143,32 @@ const updateSorting = () => {
   }
 };
 const updateQueries = async () => {
+  const inputs = mixedFarmFilters.value.inputs;
+  if (inputs && filterFarmInputs.value) {
+    const nFltrNptsVal = filterFarmInputs.value;
+    if (nFltrNptsVal.farmId) {
+      inputs.farmId.value = nFltrNptsVal.farmId.value;
+    }
+    if (nFltrNptsVal.name) {
+      inputs.name.value = nFltrNptsVal.name.value;
+    }
+    if (nFltrNptsVal.freeIps) {
+      inputs.freeIps.value = nFltrNptsVal.freeIps.value;
+    }
+  }
   const options = mixedFarmFilters.value.options;
   if (options) {
     options.page = page.value;
     options.size = size.value;
     options.sortBy = sortBy.value;
   }
+  await updateFarms();
 };
-watch(mixedFarmFilters.value, updateFarms, { deep: true });
 
+watch(mixedFarmFilters.value, updateFarms, { deep: true });
```

Comment:
We can watch on the reference, not the value, because you used {deep: true} -> which means watch on it deep, please

```suggestion
watch(mixedFarmFilters, updateFarms, { deep: true });
```